src/errors.rs

```diff
@@ -86,7 +86,14 @@ impl<T> std::error::Error for SendError<T> {}
 /// before sending any message, or if a message has already been received on the channel.
 #[cfg(any(feature = "std", feature = "async"))]
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
```

Comment:
Maybe consider removing `Copy` too? Otherwise it'd be harder to expand with containing e.g. a `String`.

Reply:
Good catch! Thanks. The `Copy` should definitely go away.